pages/src/App.js

```diff
@@ -48,6 +48,32 @@ class App extends Component {
 
   _onEditTodo = this._onCreateTodo
 
+  _onDeleteElementToDo = id => {
+
+  fetch(`/todos/${id}`, {
```

Comment:
Be careful of the coding style: no unnecessary empty line like above and correctly indent according to the nesting level.

Reply:
okay